UPGRADING.rst

```diff
@@ -66,3 +63,17 @@ Examples:
 +-------------+---------------+-----------+-------------+
 | ``foo-bar`` | ``-``         | 2         | 2           |
 +-------------+---------------+-----------+-------------+
+
+Graylog REST API
+================
+
+Streams API
+-----------
+
+Due to the introduction of index sets, the payload for creating, updating and cloning of streams now requires the ``index_set_id`` field. The value for this needs to be the ID of an existing index set.
+
+Affected endpoints:
+
+* ``POST /streams``
+* ``PUT  /streams/{streamId}``
```

Comment:
In `UpdateStreamRequest` `index_set_id` is nullable, do we want to be overly specific here or not?
Or is that a bug because the other two require the field.